app/Http/Controllers/LoginCredentialController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\{
+    User,
+    LoginCredential
+};
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+use App\Http\Requests\LoginCredentialRequest;
+
+class LoginCredentialController extends Controller
+{
+    public function create(Request $request)
+    {
+        return view('login_credential.create');
+    }
+
+    public function store(LoginCredentialRequest $request)
+    {
+        if(!is_null($request->session('login_session_token'))){
+            $request->session()->forget('login_session_token');
+        }
+
+        $is_duplicated_login_session_token = true;
+        while($is_duplicated_login_session_token){
+            $login_session_token = Str::random(rand(30,50));
```

Comment:
```suggestion
            $login_session_token = Str::random(rand(30, 50));
```
カンマの次は半角スペース一つ空けてください。

Reply:
ご指摘ありがとうございます。
修正致しましたのでご確認よろしくお願い致します。

app/Http/Controllers/LoginCredentialController.php